### PR TITLE
Benefits component

### PR DESCRIPTION
**Pull request by Anna-MariiaPetryk**
# PR Details
Benefits component implementation for the landing page.

### CODE REVIEW

File: pages/views/_data/site.json [hunk 1]
```diff
@@ -4,5 +4,37 @@
   "github": {
     "url": "https://github.com/exadel-inc/esl",
     "srcUrl": "https://github.com/exadel-inc/esl/tree/main/"
-  }
+  },
+  "benefit" : [
```
Comment (abarmina) [inline pages/views/_data/site.json:8]: I suggest moving this to a separate file in the `_data` folder

File: pages/views/_layouts/landing.njk [hunk 2]
```diff
@@ -15,6 +15,8 @@
       {% endif %}
       {{ content | safe or 'No content.' }}
     </div>
+    {% from 'benefit.njk' import benefit with context %}
+    {{ benefit ( site.benefit ) }}
```
Comment (abarmina) [inline pages/views/_layouts/landing.njk:19]: mb can we immediately use {% for %} here (whole file benefit.njk)? without macro

File: pages/src/benefit-component/benefit-component.less [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+@media @lg {
+  .benefit-container {
+      justify-content: space-between;
+    }
+}
+
+@media @xl {
+  .benefit{
```
Comment (abarmina) [inline pages/src/benefit-component/benefit-component.less:55]:
```suggestion
  .benefit {
```

**Review by ala-n — APPROVED** [review]

**Review by dshovchko — APPROVED** [review]

File: pages/src/benefit-component/benefit-component.less [hunk 4]
```diff
@@ -0,0 +1,65 @@
+@import (reference) "../common/variables.less";
+
+.benefit {
+  color: #fff;
```
Comment (dshovchko) [inline pages/src/benefit-component/benefit-component.less:4]: Just a question, isn't there a way to define colors through variables? After all, I've heard there will be light and dark themes. And the case of specifying a specific value in such a case will play a nasty trick.

Comment (abarmina) [reply pages/src/benefit-component/benefit-component.less:4]: I think we can put it in this variable file `src/common/variables.less`

Comment (ala-n) [reply pages/src/benefit-component/benefit-component.less:4]: Text color should be cascading, so we will need to have it defined on the parent container ideally.

**Pull request merged**